package.json

```diff
@@ -23,7 +23,8 @@
     "@opentelemetry/api": "^1.2.0",
     "@opentelemetry/auto-instrumentations-node": "^0.33.1",
     "@opentelemetry/sdk-node": "^0.33.0",
-    "@tanstack/react-query": "^4.9.0",
+    "@tanstack/react-query": "^4.10.1",
+    "@tanstack/react-query-devtools": "^4.10.1",
```

Comment:
It seems like this can be put into `devDependencies` (--save-dev) and the npm build/start will still work. 
I'd be a little happier with that as node shouldn't package it at all for production (risk reduction).

Reply:
I don't think it make sense to save it as dev dependency because it is used in the application and it's not a dev tool that help to build the application.

https://tanstack.com/query/v4/docs/devtools
> By default, React Query Devtools are only included in bundles when process.env.NODE_ENV === 'development', so you don't need to worry about excluding them during a production build.

Reply:
Just because the package promises to not execute in production, doesn't mean they won't have bugs or be susceptible to attacks - for every package we include in our production build, we are also susceptible to all the security issues (despite the controls in place). As this package is designed to only be used in development; it should only be package for development. This helps to reduce our security risk for something that is unnecessarily needed in a production build.